core/src/test/java/org/opensearch/sql/data/model/ExprValueCompareTest.java

```diff
@@ -152,12 +111,12 @@ private static Stream<Arguments> getNotEqualDatetimeValuesOfDifferentTypes() {
   public void compareNeqDifferentDateTimeValueTypes(ExprValue left, ExprValue right) {
     assertNotEquals(
         0,
-        extractDateTime(left, functionProperties)
-            .compareTo(extractDateTime(right, functionProperties)));
+        extractTimestamp(left, functionProperties)
+            .compareTo(extractTimestamp(right, functionProperties)));
```

Comment:
duplicate

Reply:
removed in https://github.com/opensearch-project/sql/pull/1980/commits/ea2a96af852e7c97a82e31bf016704466dbb4a72